mods/persistence/controllers/subsystems/persistence.dm

```diff
@@ -27,6 +27,10 @@ SUBSYSTEM_DEF(persistence)
 	// Collect the z-levels we're saving and get the turfs!
 	to_world_log("Saving [LAZYLEN(SSpersistence.saved_levels)] z-levels. World size max ([world.maxx],[world.maxy])")
 	var/start = world.timeofday
+	
+	// Launch events
+
+	GLOB.world_saving_start_event.raise_event(src)
```

Comment:
Why was this moved to before the wipe? Out of curiosity.

Reply:
Just for consistency's sake, as the end event occurs outside of the try/catch. Neutral on it though, and it can be moved back if it makes more sense to you.